util/humansize.c

```diff
@@ -67,7 +67,13 @@ humansize_parse(const char * s, uint64_t * size)
 	int state = 0;
 	uint64_t multiplier = 1;
 
-	do {
+	/* Special case for "". */
```

Comment:
I'm not convinced that this is the best way to handle it.  I tried to work it into the main loop, but putting an `if (*s == '\0')` inside the `case 0` didn't seem any better.  Although I suppose then the loop could stay as a `do... while`.